{
  "uid" : "bb0033902a552cf8",
  "name" : "test_get_css_property_footer_license[font-size-16px-\\u0440\\u0430\\u0437\\u043c\\u0435\\u0440\\u0430 \\u0448\\u0440\\u0438\\u0444\\u0442\\u0430-https://pulsewave.ru/auth/password?email=-chrome]",
  "fullName" : "tests.test_footer.test_footer.TestFooter#test_get_css_property_footer_license",
  "historyId" : "d669d67cef8b7cd9da94494ccf2e59cc",
  "time" : {
    "start" : 1714443550464,
    "stop" : 1714443550464,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x5570b549ae33 <unknown>\n#1 0x5570b518a567 <unknown>\n#2 0x5570b5182879 <unknown>\n#3 0x5570b5172852 <unknown>\n#4 0x5570b5173e44 <unknown>\n#5 0x5570b5172bbe <unknown>\n#6 0x5570b51722e0 <unknown>\n#7 0x5570b5172181 <unknown>\n#8 0x5570b51700ec <unknown>\n#9 0x5570b517060a <unknown>\n#10 0x5570b518ceef <unknown>\n#11 0x5570b5216695 <unknown>\n#12 0x5570b51f7362 <unknown>\n#13 0x5570b5215ae2 <unknown>\n#14 0x5570b51f7103 <unknown>\n#15 0x5570b51c81ba <unknown>\n#16 0x5570b51c8b8e <unknown>\n#17 0x5570b545f59b <unknown>\n#18 0x5570b546334b <unknown>\n#19 0x5570b544b681 <unknown>\n#20 0x5570b5463eb2 <unknown>\n#21 0x5570b54306af <unknown>\n#22 0x5570b548a428 <unknown>\n#23 0x5570b548a600 <unknown>\n#24 0x5570b5499f84 <unknown>\n#25 0x7f78d4694ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"efbb1479b54e2142ad782395b796d0d4\")>\nurl = 'https://pulsewave.ru/auth/password?email='\n\n    @pytest.fixture()\n    def footer_open(driver, url):\n        page = FooterPage(driver)\n>       driver.get(url)\n\ntests/test_footer/conftest.py:8: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:356: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:347: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f17c7becf50>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5570b548a600 \\\\u003Cunknown>\\\\n#24 0x5570b5499f84 \\\\u003Cunknown>\\\\n#25 0x7f78d4694ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome-headless-shell=124.0.6367.60)\nE       Stacktrace:\nE       #0 0x5570b549ae33 <unknown>\nE       #1 0x5570b518a567 <unknown>\nE       #2 0x5570b5182879 <unknown>\nE       #3 0x5570b5172852 <unknown>\nE       #4 0x5570b5173e44 <unknown>\nE       #5 0x5570b5172bbe <unknown>\nE       #6 0x5570b51722e0 <unknown>\nE       #7 0x5570b5172181 <unknown>\nE       #8 0x5570b51700ec <unknown>\nE       #9 0x5570b517060a <unknown>\nE       #10 0x5570b518ceef <unknown>\nE       #11 0x5570b5216695 <unknown>\nE       #12 0x5570b51f7362 <unknown>\nE       #13 0x5570b5215ae2 <unknown>\nE       #14 0x5570b51f7103 <unknown>\nE       #15 0x5570b51c81ba <unknown>\nE       #16 0x5570b51c8b8e <unknown>\nE       #17 0x5570b545f59b <unknown>\nE       #18 0x5570b546334b <unknown>\nE       #19 0x5570b544b681 <unknown>\nE       #20 0x5570b5463eb2 <unknown>\nE       #21 0x5570b54306af <unknown>\nE       #22 0x5570b548a428 <unknown>\nE       #23 0x5570b548a600 <unknown>\nE       #24 0x5570b5499f84 <unknown>\nE       #25 0x7f78d4694ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1714443550465,
      "stop" : 1714443550664,
      "duration" : 199
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1714443483373,
      "stop" : 1714443483373,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1714443483355,
      "stop" : 1714443483372,
      "duration" : 17
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "footer_open",
    "time" : {
      "start" : 1714443550664,
      "stop" : 1714443550881,
      "duration" : 217
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x5570b549ae33 <unknown>\n#1 0x5570b518a567 <unknown>\n#2 0x5570b5182879 <unknown>\n#3 0x5570b5172852 <unknown>\n#4 0x5570b5173e44 <unknown>\n#5 0x5570b5172bbe <unknown>\n#6 0x5570b51722e0 <unknown>\n#7 0x5570b5172181 <unknown>\n#8 0x5570b51700ec <unknown>\n#9 0x5570b517060a <unknown>\n#10 0x5570b518ceef <unknown>\n#11 0x5570b5216695 <unknown>\n#12 0x5570b51f7362 <unknown>\n#13 0x5570b5215ae2 <unknown>\n#14 0x5570b51f7103 <unknown>\n#15 0x5570b51c81ba <unknown>\n#16 0x5570b51c8b8e <unknown>\n#17 0x5570b545f59b <unknown>\n#18 0x5570b546334b <unknown>\n#19 0x5570b544b681 <unknown>\n#20 0x5570b5463eb2 <unknown>\n#21 0x5570b54306af <unknown>\n#22 0x5570b548a428 <unknown>\n#23 0x5570b548a600 <unknown>\n#24 0x5570b5499f84 <unknown>\n#25 0x7f78d4694ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 917, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/tests/test_footer/conftest.py\", line 8, in footer_open\n    driver.get(url)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1714443550912,
      "stop" : 1714443550970,
      "duration" : 58
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Тестирование Футера"
  }, {
    "name" : "tag",
    "value" : "regress"
  }, {
    "name" : "parentSuite",
    "value" : "tests.test_footer"
  }, {
    "name" : "suite",
    "value" : "test_footer"
  }, {
    "name" : "subSuite",
    "value" : "TestFooter"
  }, {
    "name" : "host",
    "value" : "fv-az1385-609"
  }, {
    "name" : "thread",
    "value" : "2312-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_footer.test_footer"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "browser",
    "value" : "'chrome'"
  }, {
    "name" : "css_property",
    "value" : "'font-size'"
  }, {
    "name" : "figma",
    "value" : "'16px'"
  }, {
    "name" : "name",
    "value" : "'размера шрифта'"
  }, {
    "name" : "url",
    "value" : "'https://pulsewave.ru/auth/password?email='"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 11,
        "unknown" : 0,
        "total" : 14
      },
      "items" : [ {
        "uid" : "85b392c88f9ec4c8",
        "reportUrl" : "https://Nat754.github.io/PulseWave/9//#testresult/85b392c88f9ec4c8",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x56487adb0e33 <unknown>\n#1 0x56487aaa0567 <unknown>\n#2 0x56487aa98879 <unknown>\n#3 0x56487aa88852 <unknown>\n#4 0x56487aa89e44 <unknown>\n#5 0x56487aa88bbe <unknown>\n#6 0x56487aa882e0 <unknown>\n#7 0x56487aa88181 <unknown>\n#8 0x56487aa860ec <unknown>\n#9 0x56487aa8660a <unknown>\n#10 0x56487aaa2eef <unknown>\n#11 0x56487ab2c695 <unknown>\n#12 0x56487ab0d362 <unknown>\n#13 0x56487ab2bae2 <unknown>\n#14 0x56487ab0d103 <unknown>\n#15 0x56487aade1ba <unknown>\n#16 0x56487aadeb8e <unknown>\n#17 0x56487ad7559b <unknown>\n#18 0x56487ad7934b <unknown>\n#19 0x56487ad61681 <unknown>\n#20 0x56487ad79eb2 <unknown>\n#21 0x56487ad466af <unknown>\n#22 0x56487ada0428 <unknown>\n#23 0x56487ada0600 <unknown>\n#24 0x56487adaff84 <unknown>\n#25 0x7f5a4a894ac3 <unknown>",
        "time" : {
          "start" : 1714357210072,
          "stop" : 1714357210072,
          "duration" : 0
        }
      }, {
        "uid" : "3315d3f85a1a51d7",
        "reportUrl" : "https://Nat754.github.io/PulseWave/8//#testresult/3315d3f85a1a51d7",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5611253c4e33 <unknown>\n#1 0x5611250b4567 <unknown>\n#2 0x5611250ff27d <unknown>\n#3 0x5611250ff331 <unknown>\n#4 0x561125142624 <unknown>\n#5 0x56112512138d <unknown>\n#6 0x56112513fae2 <unknown>\n#7 0x561125121103 <unknown>\n#8 0x5611250f21ba <unknown>\n#9 0x5611250f2b8e <unknown>\n#10 0x56112538959b <unknown>\n#11 0x56112538d34b <unknown>\n#12 0x561125375681 <unknown>\n#13 0x56112538deb2 <unknown>\n#14 0x56112535a6af <unknown>\n#15 0x5611253b4428 <unknown>\n#16 0x5611253b4600 <unknown>\n#17 0x5611253c3f84 <unknown>\n#18 0x7f756c294ac3 <unknown>",
        "time" : {
          "start" : 1714271393561,
          "stop" : 1714271393561,
          "duration" : 0
        }
      }, {
        "uid" : "97734f2dbe482979",
        "reportUrl" : "https://Nat754.github.io/PulseWave/7//#testresult/97734f2dbe482979",
        "status" : "passed",
        "time" : {
          "start" : 1714184761160,
          "stop" : 1714184761196,
          "duration" : 36
        }
      }, {
        "uid" : "70b6435468b50503",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/70b6435468b50503",
        "status" : "passed",
        "time" : {
          "start" : 1714148270842,
          "stop" : 1714148270877,
          "duration" : 35
        }
      }, {
        "uid" : "3ee57b6d78e604c3",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/3ee57b6d78e604c3",
        "status" : "passed",
        "time" : {
          "start" : 1714098317723,
          "stop" : 1714098317757,
          "duration" : 34
        }
      }, {
        "uid" : "5b3ae7b95cbd3e41",
        "reportUrl" : "https://Nat754.github.io/PulseWave/5//#testresult/5b3ae7b95cbd3e41",
        "status" : "passed",
        "time" : {
          "start" : 1714014228197,
          "stop" : 1714014228231,
          "duration" : 34
        }
      }, {
        "uid" : "571ea45f15f02b77",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/571ea45f15f02b77",
        "status" : "passed",
        "time" : {
          "start" : 1713945710686,
          "stop" : 1713945710719,
          "duration" : 33
        }
      }, {
        "uid" : "be7215cc4849566b",
        "reportUrl" : "https://Nat754.github.io/PulseWave/4//#testresult/be7215cc4849566b",
        "status" : "passed",
        "time" : {
          "start" : 1713925504561,
          "stop" : 1713925504594,
          "duration" : 33
        }
      }, {
        "uid" : "b19447dc2b5f8242",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/b19447dc2b5f8242",
        "status" : "passed",
        "time" : {
          "start" : 1713901336500,
          "stop" : 1713901336535,
          "duration" : 35
        }
      }, {
        "uid" : "a44d308e6609f95f",
        "reportUrl" : "https://Nat754.github.io/PulseWave/210//#testresult/a44d308e6609f95f",
        "status" : "passed",
        "time" : {
          "start" : 1713885493145,
          "stop" : 1713885493178,
          "duration" : 33
        }
      }, {
        "uid" : "1044f8dd378f36c2",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/1044f8dd378f36c2",
        "status" : "passed",
        "time" : {
          "start" : 1713883403409,
          "stop" : 1713883403442,
          "duration" : 33
        }
      }, {
        "uid" : "a2fbb8df5c007357",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/a2fbb8df5c007357",
        "status" : "passed",
        "time" : {
          "start" : 1713881907588,
          "stop" : 1713881907622,
          "duration" : 34
        }
      }, {
        "uid" : "b4b6df1628048a",
        "reportUrl" : "https://Nat754.github.io/PulseWave/208//#testresult/b4b6df1628048a",
        "status" : "passed",
        "time" : {
          "start" : 1713879083134,
          "stop" : 1713879083167,
          "duration" : 33
        }
      } ]
    },
    "tags" : [ "regress" ]
  },
  "source" : "bb0033902a552cf8.json",
  "parameterValues" : [ "'chrome'", "'font-size'", "'16px'", "'размера шрифта'", "'https://pulsewave.ru/auth/password?email='" ]
}